portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,14 +47,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int count = Integer.parseInt(request.getParameter("count"));
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-      
+    
+    String languageCode = request.getParameter("languageCode");
     //Iterate through query results and add entities to List
     List<String> comments = new ArrayList<>();
     int counter = 0;
     for (Entity entity : results.asIterable()) {
         if (counter < count) {
             String comment = (String) entity.getProperty("comment");
-            comments.add(comment);
+            //Translate comment to requested language
```

Comment:
nit: whitespace after "// Translate"

any luck getting the linter to work?  See the troubleshooting doc